aptos-move/e2e-move-tests/src/tests/names.rs

```diff
@@ -16,6 +19,19 @@ use cached_packages::aptos_names_sdk_builder;
     multisig_auth_key: 0x4407b9a063ac530f8b621f7d80b527a79c626791b14b51c1118763ce941b99ce
     threshold: 1/2
 */
+
+// generate a sample signature for testing aptos_names::verify::verify_register_domain_signature()
```

Comment:
Nit: I don't think this comment is helpful/accurate... This is a struct declaration; a struct does not "generate" a sample signature.

Reply:
oh yeah good point, will update this